sda-download/api/sda/sda.go

```diff
@@ -17,8 +18,11 @@ import (
 	"github.com/neicnordic/sda-download/api/middleware"
 	"github.com/neicnordic/sda-download/internal/config"
 	"github.com/neicnordic/sda-download/internal/database"
+	re "github.com/neicnordic/sda-download/internal/reencrypt"
```

Comment:
Not wrong, but given that this reference seems to be used two times, maybe we don't need a shorthand? It may be a defect in me, but I also read `re` as regular expressions and get semiconfused each time.

Reply:
I followed the usage from https://github.com/neicnordic/sensitive-data-archive/blob/main/sda/cmd/reencrypt/reencrypt_test.go#L20. But you are right, the short name `re` might not be optimal.

Reply:
I have removed the use of shortname of `re` for reencrypt in the commit 543b2118